web/css/style.css

```diff
@@ -0,0 +1,239 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+@import "font.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto;
+  margin: 0 36px;
+}
+
+.main-title {
+  height: 40px;
+  font-size: 36px;
+  text-align: center;
+  margin: 0;
+}
+
+.second-title {
+  height: 30px;
+  font-size: 24px;
+  text-align: center;
+  margin: 0;
+}
+
+.section-title,
+.section-subtitle {
+  height: 30px;
+  margin: 0;
+  color: #808080;
+}
+
+.section-title {
+  font-size: 22px;
+}
+
+.section-subtitle {
+  font-size: 22px;
+}
+
+grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+
+grid > c {
+  /* overflow: hidden; */
+}
+
+grid > c[span="row"] {
+  /* grid-column-start: 1;
+  grid-column-end: -1; */
+  grid-column: 1 / -1;
+}
+
+c[height="30"] {
+  height: 30px;
+}
+
+.box {
+  background-color: #fafafa;
+  padding: 20px;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+}
+
+.description {
+  margin: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 30px;
+  color: #808080;
+}
+
+.bAtention {
+  background-color: #FE004E;
+  color: #fff;
+  height: 40px;
+  padding: 0 24px;
+  border-radius: 20px;
+  border: 0;
+  font-size: 12px;
+  font-weight: bold;
+}
+
+.circle {
+  background-color: #2DD070;
+  color: #fff;
+  border-radius: 50%;
+  height: 40px;
+  width: 40px;
+  border: 0;
+}
+
+
+
+.tCentered {
+  text-align: center;
+}
+
+.avatar {
+  width: 100%;
+  border-radius: 5px;
+  border: 1px solid #808080;
+}
+
+.image-wrapper {
+  border-radius: 6px;
+  padding: 8px;
+  border: 1px solid #808080;
+}
+
+
+c[span="1"] {
+  grid-column-end: span 1;
+}
+
+a[class^="bDisplay__"] {
+  width: 100%;
+  padding-top: 100%;
+  border-radius: 100%;
+  display: block;
+  position: relative;
+}
+
+a[class^="bDisplay__"] > [class^="my-cv-"] {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 8vw;
+}
+
+.bDisplay__onahau {
```

Comment:
have fun with updating the name of the selectors as well as the html attribute values

Reply:
done